messageV2_test.go

```diff
@@ -0,0 +1,369 @@
+package pglogrepl
+
+import (
+	"errors"
+	"fmt"
+	"github.com/stretchr/testify/suite"
+	"testing"
+)
+
+func putMessageTestData(msg []byte, s *messageSuite) *LogicalDecodingMessage {
```

Comment:
`func (s *messageSuite) putMessageTestData(msg []byte) *LogicalDecodingMessage` ?

Reply:
And move this method to stay with `messageSuite`?